BlackwingLair/Locales/zhCN.lua

```diff
@@ -18,9 +18,8 @@ if L then
 	L.breath = "吐息警报"
 	L.breath_desc = "吐息警报"
 
-	L.debuffs = "{23174} （{605}）" -- 多彩变形 （精神控制）
```

Comment:
My point is that there is no harm in having this code. It needs to be in the locale anyway, so why not make it proper Chinese? :)